parser/lexer.go

```diff
@@ -243,7 +243,7 @@ func startWithAt(s *Scanner) (tok int, pos Pos, lit string) {
 		tok, lit = userVar, s.r.data(&pos)
 	} else if ch1 == '@' {
 		s.r.inc()
-		stream := s.r.s[pos.Offset+2:]
+		stream := strings.ToLower(s.r.s[pos.Offset+2:])
```

Comment:
I suggest write a `hasPrefix` case insensitive function here, it has two advantages:

1. remain the raw input data as it is (error message should be benefited).
2. reduce object allocation.